README.md

```diff
@@ -41,29 +41,26 @@ Before you begin, make sure you have the following installed on your system:
     git clone https://github.com/citrineos/citrineos-core
     ```
 
-1. Navigate to the CitrineOS Server directory:
-
-    ```shell
-    cd citrineos-core/Server
-    ```
-
-1. Install project dependencies:
+1. Install project dependencies from root dir:
 
    ```shell
-   ./unix-init-install-all.sh
+   npm run install-all
    ```
 
-1. Start the server and its supporting infrastructure with:
+1. The docker container should be initialized by running `docker-compose -f ./Server/docker-compose.yml up -d` or 
+by using the IntelliJ `Server` Run Configuration which was created for this purpose.
 
-    ```shell
-    docker-compose up -d 
-    ```
+1. Running `docker-compose.yml` will ensure that the container is configured to expose the `:9229` debugging
+port for the underlying NodeJS process. A variety of tools can be utilized to establish a debugger connection
+with the exposed localhost 9229 port which is forwarded to the NodeJS service running within docker. The IntelliJ
+`Attach Debugger` Run Configuration was made to attach to a debugging session.
```

Comment:
Does this mean create a new run configuration and select "Attach to Node.js/Chrome"?

Reply:
@lydiazcheng yes I forgot to commit the run configurations, but they should be there now. CC: @venus-yu